src/routes/userRouter.ts

```diff
@@ -1,103 +1,90 @@
-import express, { Request, Response } from 'express'
-import { User } from '../../models/UserModel'
-import jwt from 'jsonwebtoken'
-import bcrypt from 'bcryptjs'
+import express, { Request, Response } from "express";
+import { User } from "models/UserModel";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcryptjs";
 
-const router = express.Router()
+const router = express.Router();
 
-router.get('/findall', async (req: Request, res: Response) => {
-  const users = await User.find({})
-  return res.status(200).send(users)
-})
+router.get("/findall", async (req: Request, res: Response) => {
+  const users = await User.find({});
+  return res.status(200).send(users);
+});
 
-router.post('/add', async (req: Request, res: Response) => {
-  // console.log(req.body.name)
-  const check = await User.findOne({name:req.body.name.toLowerCase()})
-  if(!check){
-    const name = req.body.name.toLowerCase();
```

Comment:
no need to add .toLowerCase() add validation in schema so it will always store name in lowercase